### PR TITLE
Proposal - compact card layout

### PR DESCRIPTION
**Pull request by anod**
This proposal PR adds another style for cards - compact cards, see screenshot
screen

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -443,6 +443,6 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "feeder"
+rootProject.name = "Feeder"
```
Comment (spacecowboy) [inline settings.gradle.kts:446]: This doesn't seem relevant to this PR.

Rename your directory to `feeder` instead?

Maybe I can rename the repo or something..

**Review by spacecowboy — CHANGES_REQUESTED** [review]
This looks good to me, except for the second commit (see comment). Just drop that and it can be merged IMO.

File: settings.gradle.kts (hunk 1 above)
Comment (anod) [reply settings.gradle.kts:446]: Renaming the folder to lowercase didn't work. The app won't sync for me without it. will drop for this change, but I need to apply it for myself to build the app

image)

**Review by spacecowboy — APPROVED** [review]

**Comment by spacecowboy** [comment]
Thanks again @anod ! Great work

**Pull request merged**